client/ayon_core/hosts/blender/api/render_lib.py

```diff
@@ -91,115 +100,200 @@ def set_render_format(ext, multilayer):
         image_settings.file_format = "TIFF"
 
 
-def set_render_passes(settings):
-    aov_list = (settings["blender"]
-                        ["RenderSettings"]
-                        ["aov_list"])
-
-    custom_passes = (settings["blender"]
-                             ["RenderSettings"]
-                             ["custom_passes"])
+def set_render_passes(settings, renderer):
+    aov_list = settings["blender"]["RenderSettings"]["aov_list"]
```

Comment:
```suggestion
    aov_list = set(settings["blender"]["RenderSettings"]["aov_list"])
```

You are doing so many look ups into this list you're better of making it a `set` :)